bin/g5-start.sh

```diff
@@ -26,15 +27,22 @@ elif [ $(bc <<< "$numArgs > 0") -eq 1 ]; then
   # optional parameter $1 to specify how many minutes ago for sensor insert/start
   minutes_ago=$1
 fi
- 
-
-# check Logger last state json file to get txId in order to determine if using g6 or not
-#LAST_STATE="${HOME}/myopenaps/monitor/xdripjs/Logger-last-state.json"
-#txId=$(cat ${LDIR}/Logger-last-state.json | jq -M '.[0].txId')
-#txId="${txId%\"}"
-#txId="${txId#\"}"
-#echo ""
 
+#update config
+if [ -n $txId ]; then
+  if [ "$txId" != "null" -a "$txId" != "" ]; then  
+    cgm-transmitter "$txId"
+  fi
+fi
```

Comment:
Did you want to update the txId to the config file in this case? I'm unsure sure if that is the way to go or not ... just wondering.

Reply:
Yes, my thought was that I'll typically change my transmitter as part of swapping out a sensor so why not allow someone to specify a new transmitter id as part of cgm-start.  It may not have been clear, but I added a new -t flag to the cgm-start keeping with the current design with -m and -c flags.